wallet/storage/src/lib.rs

```diff
@@ -77,3 +83,17 @@ pub trait Transactional<'t> {
 }
 
 pub trait WalletStorage: WalletStorageWrite + for<'tx> Transactional<'tx> + Send {}
+
+impl Store<storage_sqlite::Sqlite> {
+    /// Create a default storage (mostly for testing, may want to remove this later)
+    pub fn new_in_memory() -> crate::Result<Self> {
+        Self::new(storage_sqlite::Sqlite::new_in_memory())
+    }
+
+    pub fn new_from_path(path: PathBuf) -> crate::Result<Self> {
```

Comment:
You can use the signature

```
new_from_path<P: AsRef<Path>>(path: P)
```

This basically says "take a path from anything that can become path when calling `as_ref()`"

Reply:
I will fix this problem myself later. I'm see an issue in Backend that it doesn't take Impl as parameter. This will lead to a cascade of fixes.

Reply:
I implemented it as you proposed. There is still a `PathBuf` inside the `SqliteStorageMode` struct as a simple `Path` is unsized.